marquez_airflow/dag.py

```diff
@@ -96,22 +129,25 @@ def report_jobrun_change(self, dagrun, **kwargs):
         marquez_job_run_id = self._job_id_mapping.pop(
             JobIdMapping.make_key(dagrun.dag_id, dagrun.run_id), session)
         if marquez_job_run_id:
+            log.info(f'Found job run.',
+                     airflow_dag_id=dagrun.dag_id,
+                     airflow_run_id=dagrun.run_id,
+                     marquez_run_id=marquez_job_run_id,
+                     marquez_namespace=self.marquez_namespace)
+
             if kwargs.get('success'):
                 self.get_marquez_client().mark_job_run_as_completed(
                     marquez_job_run_id)
             else:
                 self.get_marquez_client().mark_job_run_as_failed(
                     marquez_job_run_id)
-        state = 'COMPLETED' if kwargs.get('success') else 'FAILED'
-        self.log_marquez_event(['job_state_change',
-                                marquez_job_run_id,
-                               state])
 
-    def log_marquez_event(self, args):
-        logging.info("\t".join(["[Marquez]",
-                                self.marquez_namespace,
-                                self.dag_id,
-                                ] + args))
+        state = 'COMPLETED' if kwargs.get('success') else 'FAILED'
```

Comment:
Can we use `RunState.COMPLETED` from marquez-python?

Reply:
The `RunState` enum was introduced recently and not part of the current release of `marquez-python`